### PR TITLE
Fixes for Filepicker test flakiness 

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
updated the assertion and also wait for a route

### CODE REVIEW

File: app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js [hunk 1]
```diff
@@ -24,7 +24,13 @@ describe("FilePicker Widget Functionality", function() {
...
       .attachFile(fixturePath);
     cy.get(commonlocators.filePickerUploadButton).click();
-    cy.get(".bp3-spinner").should("have.length", 1);
+    cy.wait("@updateLayout").should(
+      "have.nested.property",
+      "response.body.responseMeta.status",
+      200,
+    );
+    cy.wait(500);
+    cy.get("button").contains("1 files selected");
```
Comment (hetunandu) [inline app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js:33]: This is a wrong assertion. We want to test if the file picker goes into loading state and not assert the final state

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js:33]: Ok, in that case, we want the spinner to be checked? which was the case earlier. But the problem is sometimes spinner doesn't come at all.

Comment (hetunandu) [reply app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js:33]: The file to be uploaded should be bigger

Comment (NandanAnantharamu) [reply app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js:33]: ok let try that and update the assert

Comment (hetunandu) [reply app/client/cypress/integration/Smoke_TestSuite/FormWidgets/FilePicker_spec.js:34]: You need to revert this assertion here

**Review by hetunandu — APPROVED** [review]

**Pull request merged**